src/main.js

```diff
@@ -1 +1,14 @@
-// Write your code here
\ No newline at end of file
+// Write your code here
+
+let a = 5;
+let b = 7;
+console.log(a+b);
+
+console.log(a*b);
+
+const name = "Anri";
+console.log("Hello" + name + "!");
```

Comment:
აქ არ გჭირდება `!` ნიშანი
```suggestion
console.log("Hello" + name);
```